kernel/kernel-api/src/main/java/io/delta/kernel/internal/DeltaErrorsInternal.java

```diff
@@ -55,4 +58,31 @@ public static IllegalStateException logicalPhysicalSchemaMismatch(
                 + "equal the logical schema size (%s).",
             num_partition_cols, physical_size, logical_size));
   }
+
+  public static InvalidTableException catalogCommitsPrecedePublishedDeltas(
+      String tablePath, long earliestCatalogCommitVersion, List<Long> foundPublishedVersions) {
+    return new InvalidTableException(
+        tablePath,
+        String.format(
+            "Missing delta file: found staged ratified commit for version %s but no published "
+                + "delta file. Found published deltas for versions: %s",
+            earliestCatalogCommitVersion, foundPublishedVersions));
+  }
+
+  public static InvalidTableException publishedDeltasAndCatalogCommitsNotContiguous(
+      String tablePath, List<Long> publishedDeltaVersions, List<Long> catalogCommitVersions) {
+    return new InvalidTableException(
+        tablePath,
+        String.format(
+            "Missing delta files: found published delta files for versions %s and staged "
```

Comment:
I think this should say the word "contiguous" somewhere, right? i.e. the error message should reflect the error method name